python/tests/test_tables.py

```diff
@@ -317,7 +317,7 @@ def test_set_column_attributes_errors(self):
         # Try to read a column that isn't there. (We can always write to new attributes
         # in Python, so there's nothing to test in that case.)
         with self.assertRaises(AttributeError):
-            _ = table.no_such_column
+            _ = table.no_such_column  # NOQA
```

Comment:
Maybe you've got an older version of flake8?

Reply:
Hmph, I must.